protocol/operation.re

```diff
@@ -66,21 +74,16 @@ module Side_chain = {
     nonce: int32,
     block_height: int64,
     source: Wallet.t,
```

Comment:
Question, does source makes sense for add and remove validators? When will the block producer add an Add and Remove validator operation?

Reply:
I think it's useful for bookkeeping. It's good to know who proposed that a certain validator be added or removed

Reply:
Agreed